riva/clients/nmt/riva_nmt_streaming_s2s_client.cc

```diff
@@ -162,7 +168,8 @@ main(int argc, char** argv)
       FLAGS_target_language_code, FLAGS_profanity_filter, FLAGS_automatic_punctuation,
       /* separate_recognition_per_channel*/ false, FLAGS_chunk_duration_ms, FLAGS_simulate_realtime,
       FLAGS_verbatim_transcripts, FLAGS_boosted_words_file, FLAGS_boosted_words_score,
-      FLAGS_tts_encoding, FLAGS_tts_audio_file, FLAGS_tts_sample_rate, FLAGS_tts_voice_name);
+      FLAGS_tts_encoding, FLAGS_tts_audio_file, FLAGS_tts_sample_rate, FLAGS_tts_voice_name,
+      FLAGS_tts_prosody_rate, FLAGS_tts_prosody_pitch, FLAGS_tts_prosody_volume);
```

Comment:
it will be good to add some allowed values of rate/pitch/volume for the user to set from. We add those in the help section and also add some error handling if user doesn't provide a value from those (similar to how it is done for other params below)